### DISCUSSION TITLE
TODO

### DISCUSSION THREAD

**Discussion by MLAcookie** [MLAcookie/MAA1999WPF#8] [Ideas]
随着MAA框架的改进，WPF界面开发也需要跟进
现在需要解决的：

- [ ] 兼容新的Pipeline config

之后的改进：

* 美化界面（可能会使用[FluentWPF](https://github.com/sourcechord/FluentWPF) ）
* 加入一些自动化（一键启动模拟器，定时器）
* 软件内更新资源

画大饼：

* C#底层对maa框架的集成
* 做低代码流程编辑器（有可能用Avalonia实现，当然更可能没时间写）

欢迎各位讨论

**Comment by KaronGH** [comment]
界面的话可以看看[HandyControl](https://github.com/HandyOrg/HandyControl)，MAA同款控件库。不过我是在想趁着现在GUI不是特别复杂的时候直接用Avalonia重写（如果有Avalonia执念的话）

**Reply by MLAcookie** [reply]
Avalonia只是我正在尝试的东西（主要是WPF跨平台不好搞），也不是现在着急考虑的东西。况且后端都是通用的，到时候迁移的话问题应该不会很大

**Reply by KaronGH** [reply]
嗯，主要是我也感兴趣Avalonia（未来干活可能用得到）所以我也在考虑借着这个机会转过去。M9A本体的更新频率相对不快，我觉得如果要迁移的话应该不会花太久时间
看你这边吧，我可以PR WPF版的同时私底下同步后端维护一个Avalonia版本，不过我既不懂WPF又不懂Ava所以进度估计不会快就是了2333

**Reply by 1m188** [reply]
倒也不必私底下维护，如果确定后面要转Avalonia的话不妨直接开个新的分支，两个分支同步推进，等后面新分支的内容追上来了之后可以用新分支代替原来的master

**Reply by MLAcookie** [reply]
那就直接开新的分支吧

**Comment by KaronGH** [comment]
现在有想法去MAA那边找点新功能补到这边，这项工作可能会很漫长（也有可能会咕）
